airbyte-integrations/bases/base-normalization/Dockerfile

```diff
@@ -22,5 +22,5 @@ RUN dbt deps
 WORKDIR /airbyte
 ENTRYPOINT ["/airbyte/entrypoint.sh"]
 
-LABEL io.airbyte.version=0.1.29
+LABEL io.airbyte.version=0.1.30
```

Comment:
already published... :)

Reply:
haha unlucky timing. I'll merge in master and bump before merging, assuming this is approved.